packages/components/psammead-useful-links/README.md

```diff
@@ -0,0 +1,78 @@
+
+## ⛔️ This is an alpha component ⛔️
+
+This component is currently tagged as alpha and is not suitable for production use. Following the passing of an accessibility review this component will be marked as ready for production and the alpha tag removed.
+
+## Description
+
+The `UsefulLinks` component implements a Link Promo type that accesses links with information about various things to do with the world service website. `UsefulLinks` consists of a wrapper and a link element when there is only one useful item. In case there are multiple items, `UsefulLinks` consists of an unordered list with list items.
+
+## Installation
+
+```jsx
+npm install @bbc/psammead-useful-links
+```
+
+## Props
+
+| Argument  | Type | Required | Default | Example |
+| --------- | ---- | -------- | ------- | ------- |
+| service | string | yes | N/A | `'news'` |
+| script | object | yes | latin | { canon: { groupA: { fontSize: '28', lineHeight: '32',}, groupB: { fontSize: '32', lineHeight: '36', }, groupD: { fontSize: '44', lineHeight: '48', }, }, trafalgar: { groupA: { fontSize: '20', lineHeight: '24', }, groupB: { fontSize: '24', lineHeight: '28', }, groupD: { fontSize: '32', lineHeight: '36', }, }, } |
+| usefulItems | array | yes | N/A | ['Mitocinmu da sauko da sautin labarai', 'Labaran BBC Hausa a text', 'Abokan huldar BBC Hausa'] |
+
+## Usage
+
+<!-- Description of the component usage -->
+
+```jsx
+import { latin } from '@bbc/gel-foundations/scripts';
+import {
+  UsefulLink,
+  UsefulLinksLi,
+  UsefulLinksUl,
+  UsefulLinkItem,
+} from './index';
+
+const SingleUsefulItem = () => (
+    <UsefulLink script={latin} service="news" usefulItems={["Labaran BBC Hausa a text"]} />
+);
+
+const MultipleUsefulItems = () => (
```

Comment:
Do you mind updating these examples to reflect the latest changes?

Reply:
Done:)